.docs/angular-meteor/packages/tutorial-step-diff-compiler/.npm/plugin/mdg:tutorial-step-diffs-plugin/npm-shrinkwrap.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "am-git-patch-parser": {
-      "version": "0.2.1"
+      "version": "http://registry.npmjs.org/am-git-patch-parser/-/am-git-patch-parser-0.2.1.tgz"
```

Comment:
I don't know why exactly, my linux os change that automatically i guess.
I change that, i can provide a new pr if you want. Let me know.